src/GraphQL.php

```diff
@@ -306,10 +313,19 @@ protected function loadPersistedQuery(string $sha256hash): DocumentNode
             );
         }
 
+        if (in_array($sha256hash, $this->persistedQueries)) {
+            return $this->persistedQueries[$sha256hash];
+        }
+
         $cacheFactory = Container::getInstance()->make(CacheFactory::class);
         $store = $cacheFactory->store($cacheConfig['store']);
+        $persistedQuery = $store->get("lighthouse:query:{$sha256hash}");
+
+        if ($persistedQuery) {
+            $this->persistedQueries[$sha256hash] = $persistedQuery;
```

Comment:
I think this will leak memory when used with Laravel Octane. The cache should be pretty fast, and the method expresses that it is reaching out to something external with the verb `load` in its name, so users should be aware.